src/index.html

```diff
@@ -3,13 +3,12 @@
 <head>
     <meta charset="UTF-8">
     <title></title>
-    <link rel="stylesheet" href="css/style.css"/>
-    <script src="js/bundle.js"></script>
```

Comment:
Manual inclusion of bundles are not needed. HtmlWebpackPlugin will generate those script entries.